### PR TITLE
Add engines, answers endpoints, refactor axios requests.

### LINKED ISSUE #23: Add files, answers, classifications, engines endpoints
I'm making a [GraphQL API](https://github.com/anonrose/gpt3.graphql) using this lib it would be great to have the files, answers, classifications, engines endpoints available as well, I will open a PR shortly with the changes but wanted to open an issue first.

### PR DESCRIPTION
**Pull request by anonrose**
resolves #23

### CODE REVIEW

**Comment by anonrose** [comment]
addresses engines, answers from https://github.com/Njerschow/openai-api/issues/23 I will add classifications and files in a later PR

File: index.js [hunk 1]
```diff
@@ -1,69 +1,67 @@
...
 
-    _send_request(opts) {
-        const url = config.completionURL(opts.engine);
-        const reqOpts = {
-            headers: {
-                'Authorization': `Bearer ${this._api_key}`,
-                'Content-Type': 'application/json'
-            }
-        };
-        const data = Object.assign({},
```
Comment (Cormanz) [inline index.js:27]: This seems important, why did you remove it? People should be using `maxTokens` and not `max_tokens` with `openai-api`, just like the npm page shows.

Comment (anonrose) [reply index.js:27]: You still do use `maxTokens`, the camel case gets converted to snake case [here](https://github.com/Njerschow/openai-api/pull/24/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R18-R20)

Comment (Cormanz) [reply index.js:27]: Oh, okay

File: test/basic.test.js [hunk 2]
```diff
@@ -1,60 +1,66 @@
...
-        })
-        .catch(err => {
-            console.error(err);
-            expect.fail();
-        });
+  this.timeout(6000);
+
+  const openai = new OpenAI(api_key);
+
+  it('should handle simple completion', async function () {
```
Comment (Cormanz) [inline test/basic.test.js:15]: Why not use `async () => {}` over `async function() {}`?

Comment (anonrose) [reply test/basic.test.js:15]: While it shortens the syntax, there's no use of `this` to access any outside context so it's a moot change.

**Comment by anonrose** [comment]
@Njerschow can you please review/merge this? Really wanting to wrap up a dependent project.

**Comment by Cormanz** [comment]
@anonrose for the time being with npm you could just use `npm i github:anonrose/openai-api`

**Review by Njerschow — APPROVED** [review]
looks good

**Comment by Njerschow** [comment]
Closes #23
